tests/Avalonia.RenderTests/Media/ImageBrushTests.cs

```diff
@@ -19,15 +19,9 @@ public ImageBrushTests()
         {
         }
 
-        private string BitmapPath
-        {
-            get { return System.IO.Path.Combine(OutputPath, "github_icon.png"); }
-        }
+        private string BitmapPath => System.IO.Path.Combine(OutputPath, "github_icon.png");
```

Comment:
IMO this is a step too far. It's less easy to read. The below would have been enough. Yes, this is dev preference which is why I would change less actually. Other people will have different opinions I'm sure.

```
        private string BitmapPath
        {
            get => System.IO.Path.Combine(OutputPath, "github_icon.png");
        }
```